Marlin/Configuration.h

```diff
@@ -950,9 +950,16 @@
 
 /**
  * Z Servo Probe, such as an endstop switch on a rotating arm.
+ *
+ * The Z_SERVO_MEASURE_ANGLE is only useful if your servo needs to
+ * move to a "free" position to allow measuring.
+ * With Z_SERVO_INTERMEDIATE_STOW you can have the probe stowed
+ * between different points of a G29.
  */
-//#define Z_PROBE_SERVO_NR 0       // Defaults to SERVO 0 connector.
-//#define Z_SERVO_ANGLES { 70, 0 } // Z Servo Deploy and Stow angles
+//#define Z_PROBE_SERVO_NR 0          // Defaults to SERVO 0 connector.
+//#define Z_SERVO_ANGLES { 70, 0 }    // Z Servo Deploy and Stow angles
+//#define Z_SERVO_MEASURE_ANGLE 45    // Z Servo Angle after deploy (to allow measuring)
+//#define Z_SERVO_INTERMEDIATE_STOW   // Stow the probe between points
```

Comment:
There is a flag to `G29` which already does a stow after every probe, so there's no need to make a new "always on" behavior. Instead, just use that form of `G29` with your specific setup.

Reply:
> There is a flag to `G29` which already does a stow after every probe, so there's no need to make a new "always on" behavior. Instead, just use that form of `G29` with your specific setup.

As far as I understand this would leave it to the user (or slicer) to specify the E option on G29, whereas with some probes you might want to do this by default to avoid the risk of damaging a fragile probe, no?  Adding it as compile time option was largely inspired by the SLOW vs HIGH SPEED modes of the BLTOUCH code, with the default slow mode also retracting without having to specify the E option to G29.

Reply:
Once the firmware is built, the behavior is fixed. I'm not sure how an obscure and brand new config option will be more obvious to users than using the long-existing and well-documented feature to accomplish the needed effect — a full stow between points — which was created with servo probes in mind in the first place.

First, you may want to check whether `G29 E` accomplishes the needed behavior just as well as this new option. And if that is the case, one possible "correct" implementation would be to make `G29 E1` the default, with `G29 E0` being required to not-stow between points. This way the option is at least tied to the existing system and doesn't introduce Yet Another Highly Specialized Option into the pile.